src/components/search/Search.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { Configuration, GeoJSONResource, JSONResource, Resource } from '@/schemas/configuration-schema';
+
+import { Typeahead } from 'react-bootstrap-typeahead';
+import { createSharedPathnamesNavigation } from 'next-intl/navigation';
+import { locales } from '@/locales';
+import { useMiniSearch } from 'react-minisearch';
+import { useState } from 'react';
+import { useTranslations } from 'next-intl';
+
+const { useRouter } = createSharedPathnamesNavigation({
+  locales: [...locales.values()],
+});
+
+export default function Search({ configuration, className }: { configuration: Configuration; className?: string }) {
+  const { search, searchResults } = useMiniSearch(configuration.resources, { fields: ['id', 'name', 'description'] });
+  const t = useTranslations('Search');
+  const [selected, setSelected] = useState([] as (object | string)[]);
+  const router = useRouter();
+
+  return (
+    <Typeahead
+      className={className}
+      filterBy={() => true}
+      id="search"
+      labelKey="name"
+      highlightOnlyResult
+      onInputChange={(term) => {
+        search(term, { prefix: true });
```

Comment:
I played around with the search a bit, and I noticed that when I search "Points of Interest", the search only suggest 1 entry, but once I type "Points of Interest K", I also get suggested the "Karlsruher Mietspiegel" pdf. I believe we should mitigate this by adding `combineWith: 'AND'` to the search settings. I also think we should add `fuzzy: 0.2`, to account for minor typos.

Reply:
Very good points. I've set `fuzzy` to `2` which equals to typos of length 1. After playing around with it some more I think this is pretty good.